InternshipProgram/VeryEasy/VeryEasyProgram.cs

```diff
@@ -9,47 +9,93 @@ public static class VeryEasyProgram
         /// </summary>
         public static bool ReturnTrue()
         {
-            throw new NotImplementedException();
+            return true;
         }
         
         /// <summary>
         /// Create a function that takes two numbers as arguments and return their sum
         /// </summary>
         public static int Sum(int a, int b)
         {
-            throw new NotImplementedException();
+            int sum = a + b;
+
+            return sum;
         }
         
         /// <summary>
         /// Write a function that takes an integer minutes and converts it to seconds
         /// </summary>
         public static int ConverMinutesToSeconds(int minutes)
         {
-            throw new NotImplementedException();
+            int seconds = minutes * 60;
+            
+            return seconds;
         }
         
         /// <summary>
         /// Create a function that returns true if an integer is evenly divisible by 5, and false otherwise
         /// </summary>
         public static bool DivisibleByFive(int number)
         {
-            throw new NotImplementedException();
+            int rest = number % 5;
+
+            if (rest == 0)
+            {
+                return true;
+            }
+
+            return false;
         }
         
         /// <summary>
         /// Create a function that takes two numbers and a mathematical operator and returns the result
         /// </summary>
         public static int Calculate(int num1, int num2, string operation)
         {
-            throw new NotImplementedException();
+            if (operation == "+")
+            {
+                int result = num1 + num2;
+                return result;
+            } 
+            else if (operation == "-")
+            {
+                int result = num1 - num2;
+                return result;
+            }
+            else if (operation == "*")
+            {
+                int result = num1 * num2;
+                return result;
+            }
+            else if (operation == "/")
+            {
+                int result = num1 / num2;
+                return result;
+            }
+            else
+            {
+                int result = num1 % num2;
+                return result;
+            }
         }
         
         /// <summary>
         /// Write a function to reverse an array
         /// </summary>
         public static int[] Reverse(int[] array)
         {
-            throw new NotImplementedException();
+            int[] reverse = new int[array.Length];
+            int count = 0;
+            int i = array.Length - 1;
+
+            while (count < array.Length)
+            {
+                reverse[count] = array[i];
+                count++;
+                i--;
+            }
+
+            return reverse;
```

Comment:
Great resolution!
[Here's](https://docs.microsoft.com/en-us/dotnet/api/system.array.reverse?view=net-6.0)  a reference to the method.